### PR TITLE
Update docker-compose file postgres port

### PR DESCRIPTION
**Pull request by kabaros**
Update docker-compose file to change the port for Postgres so that it does not conflict if there is another instance of postgres already installed

### CODE REVIEW

File: config/database.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = ({ env }) => {
         settings: {
           client: 'postgres',
           host: env('DATABASE_HOST', '127.0.0.1'),
-          port: env.int('DATABASE_PORT', 5432),
+          port: env.int('DATABASE_PORT', 5433),
```
Comment (abolkog) [inline config/database.js:10]: Why did you change it ?

**Review by abolkog — CHANGES_REQUESTED** [review]

File: config/database.js (hunk 1 above)
Comment (kabaros) [reply config/database.js:10]: rtfm

**Review by abolkog — APPROVED** [review]
LGTM
